routes/api/contacts.js

```diff
@@ -1,25 +1,76 @@
-const express = require('express')
+const Joi = require("joi");
+const express = require("express");
+const router = express.Router();
+const contacts = require("../../models/contacts");
 
-const router = express.Router()
+const schema = Joi.object({
+  name: Joi.string().min(3).max(20),
+  email: Joi.string().email({ minDomainSegments: 2 }),
+  phone: Joi.string().min(9),
```

Comment:
Sprawdź czy phone moze być pominięty? Jeśli tak, to trzeba dodać require do każdego z tych pól.